From ecab161e2bf566f1e2855078a4a26a5e63f31b20 Mon Sep 17 00:00:00 2001
From: Kent Russell <kent.russell@amd.com>
Date: Tue, 20 Sep 2016 11:30:07 -0400
Subject: [PATCH 1649/1722] drm/amdkfd: Remove warning when CU Mask is zero

This is a normal condition, so don't flood the dmesg with warnings for
an expected use-case.

Change-Id: I6e650774de071847a4aee4a9fa9c696de01bbdc6
Signed-off-by: Kent Russell <kent.russell@amd.com>
---
 drivers/gpu/drm/amd/amdkfd/kfd_mqd_manager_cik.c | 2 +-
 drivers/gpu/drm/amd/amdkfd/kfd_mqd_manager_vi.c  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_mqd_manager_cik.c b/drivers/gpu/drm/amd/amdkfd/kfd_mqd_manager_cik.c
index f0466b6..e3638e1 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_mqd_manager_cik.c
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_mqd_manager_cik.c
@@ -51,7 +51,7 @@ static void update_cu_mask(struct mqd_manager *mm, void *mqd,
 	const uint32_t *cu_mask = q->cu_mask;
 	int se, cu_per_sh, cu_index, i;
 
-	if (WARN_ON(cu_mask_count == 0))
+	if (cu_mask_count == 0)
 		return;
 
 	m = get_mqd(mqd);
diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_mqd_manager_vi.c b/drivers/gpu/drm/amd/amdkfd/kfd_mqd_manager_vi.c
index 4a8f94c..3f380e85 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_mqd_manager_vi.c
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_mqd_manager_vi.c
@@ -51,7 +51,7 @@ static void update_cu_mask(struct mqd_manager *mm, void *mqd,
 	const uint32_t *cu_mask = q->cu_mask;
 	int se, cu_per_sh, cu_index, i;
 
-	if (WARN_ON(cu_mask_count == 0))
+	if (cu_mask_count == 0)
 		return;
 
 	m = get_mqd(mqd);
-- 
2.7.4

